website/docs/d/cur_report_definition.html.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: "aws"
+page_title: "AWS: aws_cur_report_definition"
+sidebar_current: "docs-aws_cur_report_definition"
```

Comment:
This data source documentation page is missing a sidebar link in `website/aws.erb` and the `sidebar_current` should match the correct format.